experimental/devices/epd/example_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package epd_test
+
+import (
+	"image"
+	"log"
+
+	"periph.io/x/periph/experimental/devices/epd"
+
+	"periph.io/x/periph/conn/spi/spireg"
+
+	"periph.io/x/periph/devices/ssd1306/image1bit"
+	"periph.io/x/periph/host"
+)
+
+func Example() {
+	// Make sure periph is initialized.
+	if _, err := host.Init(); err != nil {
+		log.Fatal(err)
+	}
+
+	// Use spireg SPI bus registry to find the first available SPI bus.
+	b, err := spireg.Open("")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer b.Close()
+
+	dev, err := epd.NewSPIHat(b, &epd.EPD2in13) // Display config and size
+	if err != nil {
+		log.Fatalf("failed to initialize epd: %v", err)
+	}
+
+	// Draw on it.
+	img := image1bit.NewVerticalLSB(dev.Bounds())
+	// Note: this code is commented out so periph does not depend on:
+	//    "golang.org/x/image/font"
+	//    "golang.org/x/image/font/basicfont"
+	//    "golang.org/x/image/math/fixed"
+	//
+	// f := basicfont.Face7x13
+	// drawer := font.Drawer{
+	// 	Dst:  img,
+	// 	Src:  &image.Uniform{image1bit.On},
+	// 	Face: f,
+	// 	Dot:  fixed.P(0, img.Bounds().Dy()-1-f.Descent),
+	// }
+	// drawer.DrawString("Hello from periph!")
+
+	if err := dev.Draw(dev.Bounds(), img, image.Point{}); err != nil {
+		log.Fatal(err)
+	}
+	dev.DisplayFrame() // After drawing on the display, you have to show the frame
+}
+
+func Example_other() {
```

Comment:
I'm not sure you can use a lower case for the first word, please confirm how it shows up in godoc.

There's no blockers anymore so I'll integrate but please address the comments in a follow up.